doc/Getting-started/index.md

```diff
@@ -133,6 +133,13 @@ Public DNS or IP of your instance are available from your instance description.
 
 <br>
 
+If an error occurs, and for some reason did not run the UI part of Kaa Sandbox instance, the error can be found in the logs in the Kaa Sandbox.
+All logs are on this path - /var/log/kaa. Log system in Kaa are configured in a such way:
+    - Kaa Sandbox Frame project brings their logs in kaa-sandbox.log;
+    - Kaa Avro UI brings their logs at avro-ui-sandbox.log;
+    - Kaa node brings their logs at kaa-node.log, the most important log file for understanding the current situation.
+And there are two log files on init and shutdown Kaa Node and two files on init and shutdown Kaa Sandbox.
+
```

Comment:
If your Kaa Sandbox interface does not start due to an error, you can see the error details in the Sandbox log records.

The log records are stored in the **/var/log/kaa** directory.

The log writing system is as follows:

1. Kaa Sandbox Frame project writes its logs to the kaa-sandbox.log file.
2. Kaa Avro UI writes its logs the avro-ui-sandbox.log file.0
3. Kaa node writes its logs to the kaa-node.log file.
This is the most important log file as it contains the most of the information required for an analysis.

Also, one log file generated per each initialization and each shutdown of a Kaa node and Kaa Sandbox.